### DISCUSSION TITLE
[RESOLVED] PDF version of the online document

### DISCUSSION THREAD

**Discussion by CaledoniaProject** [github/codeql#7707] [General]
Is there a PDF version of the document here? 

* https://codeql.github.com/docs/codeql-language-guides/types-in-java/
* https://github.com/github/docs

I'm trying to print all the document and read it though, but don't know how to create a PDF from the repo above.

**Comment by smowton** [comment]
Just File -> Print and using a save-to-PDF print driver works nicely for me -- in particular it knows to drop the sidebar and only print the actual page content. I don't think there's an official PDF rendering of those docs.

**Comment by CaledoniaProject** [comment]
@smowton I need to create a PDF that contains all the sub pages. I only know how to do that with gitbook, bit codeql does not seem to use it.

**Comment by CaledoniaProject** [comment]
It's done but does not look good. https://github.com/rst2pdf/rst2pdf/issues/1063
I'll stick to the online document instead.
